src/handlers/add-user/add-user-handler.ts

```diff
@@ -0,0 +1,36 @@
+import 'reflect-metadata';
+import uuidv1 from 'uuid/v1';
+import { Context, HttpRequest } from '@azure/functions';
+import { Inject, Injectable } from '@graphql-modules/di';
+import { IUserRepository, User } from '@repositories/user-repository';
+
+@Injectable()
+class AddUser {
+  constructor(@Inject('IUserRepository') private userRepository: IUserRepository) {
+    // Nothing to do here
+  }
+
+  index = async (context: Context, req: HttpRequest): Promise<void> => {
+    const user = new User({
+      // @TODO: for now using an uuid, but we want to use auth0 Ids to setup these values
+      PartitionKey: uuidv1(),
+      RowKey: uuidv1(),
```

Comment:
We don’t want to create UUIDs here. We want to get them from the request body

Reply:
Partition key and row key should not be set here since that’s a concern of the repository. Also table storage is not the right mechanism for this, blob storage is

Reply:
That's right, once auth0 is integrated we'll get this data from the request, as mentioned in the TODO.

Reply:
I honestly don't know what a blob storage is hahahaha it sound to me that is a storage for files, but that totally doesn't make any sense and I'm sure I'm wrong 😅

Reply:
I'll move the keys to the model/repository, that makes sense

Reply:
Fixed